host/windows/runcli.bat

```diff
@@ -1,17 +1,19 @@
 @ECHO OFF
 
 :: Runs JFrog CLI with the given arguments. The JFrog CLI's home directory is the extension's home directory.
-
-set HOME_DIR=%USERPROFILE%\.jfrog-docker-desktop-extension
-set LOGS_DIR=%HOME_DIR%\logs
-
-set JFROG_CLI_HOME_DIR=%HOME_DIR%
+set "HOME_DIR=%USERPROFILE%\.jfrog-docker-desktop-extension"
+set "LOGS_DIR=%HOME_DIR%\logs"
+set "JFROG_CLI_HOME_DIR=%HOME_DIR%"
 set JFROG_CLI_USER_AGENT=jfrog-docker-extension
 set JFROG_CLI_LOG_LEVEL=DEBUG
 set CI=true
 
-if not exist %LOGS_DIR% mkdir %LOGS_DIR%
+:: Create logs directory if it doesn't exist, handling paths with spaces.
+if not exist "%LOGS_DIR%" mkdir "%LOGS_DIR%"
```

Comment:
isn't the quotes here redundent? Either here or in the definitions themselves - don't think we need both...

Reply:
I have a feeling they are. because when I echoed %CONF_FILE% I needed them.
I will try without and update accordingly.

Reply:
nope didnt work